buildbot/osuosl/master/config/builders.py

```diff
@@ -2324,8 +2324,8 @@
     'builddir': 'llvm-x86_64-debian-dylib',
```

Comment:
Shall the builder be renamed to reflect its new scope?

Reply:
I thought about that, but wasn't sure exactly what to call it? llvm-project-x86_64-dylib?